### PR TITLE
Able to install from npm

### PR DESCRIPTION
**Pull request by Grawl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "logerr",
+  "version": "0.1.0",
+  "repository": "https://github.com/i-break-codes/logerr.git",
+  "main": "logger.js",
```
Comment (i-break-codes) [inline package.json]: logerr.js :) Rest is fine..

**Review by i-break-codes — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Grawl) [reply package.json]: done

**Comment by Grawl** [comment]
#11

**Comment by i-break-codes** [comment]
Hey thanks, can you resolve the conflicts please?

**Comment by i-break-codes** [comment]
Gentle reminder!

**Comment by Grawl** [comment]
Oh sorry @i-break-codes I committed right on GitHub because I have not such a time just for now. Maybe on next week or later.

**Comment by ghosh** [comment]
@i-break-codes Seems like there is just a minor conflict in the readme. Should not be much of a problem to resolve locally. Looking forward to be able to install from npm! 😀

**Comment by Grawl** [comment]
@ghosh you can use my fork for now

**Review by i-break-codes — APPROVED** [review]

**Comment by i-break-codes** [comment]
@ghosh Yea, kinda makes sense me resolving the conflicts on his behalf as it's pretty straight forward.. will do it by tomorrow. Cheers \m/

**Comment by Grawl** [comment]
Hooray!

**Comment by i-break-codes** [comment]
Done @ghosh @Grawl, thanks for contributing 👍 ... Sorry for the delay!

**Comment by Grawl** [comment]
Glad to help you!

**Pull request merged**